fastlane/Fastfile

```diff
@@ -155,20 +151,38 @@ platform :android do
             project_dir: "./android",
             task: "bundle",
             flavor: "production",
-            build_type: "Release",
-            properties: {
-                # Version code set in build.gradle not here
-                "prodKeyStorePassword" => ENV["ANDROID_PROD_KEYSTORE_PASSWORD"]
-            }
-          )
+            build_type: "Release"
+        )
         upload_to_play_store(
             release_status:"draft",
             package_name:"digital.and.murrayapps.production",
             track:"internal",
-            json_key:"./android/app/play_store_keys/pc-api-5670955761957146419-138-d5132b70bac6.json",
+            json_key:"./android/app/play_store_keys/murray-apps-9ae8e17f9aa5.json",
             mapping_paths: ["./android/app/build/outputs/mapping/productionRelease/mapping.txt"]
         )
     end
+
+    lane :update_version_number do |options|
+        version_number_string = options[:version_number_string]
+        previous_build_number = google_play_track_version_codes(
+            package_name: "digital.and.murrayapps.production",
+            track: "internal",
+            json_key: "./android/app/play_store_keys/murray-apps-9ae8e17f9aa5.json",
+        )[0]
+        new_build_number = previous_build_number + 1
+        increment_version_code(
+            gradle_file_path: "./android/app/build.gradle",
+            version_code: new_build_number
+        )
+        increment_version_name(
+            gradle_file_path: "./android/app/build.gradle",
+            version_name: version_number_string
+          )
+        # Git
+        git_commit(path: ["*.gradle"], message: "[skip ci] Version Bumps")
```

Comment:
Why skip CI for the version bump? Any harm in keeping it in?

Reply:
It will end up going in an infinite loop. This something that needs to be improved on in a follow up to improve the release process